pages/cart-page/cart-page.css

```diff
@@ -0,0 +1,72 @@
+@import url('/main.css');
+@import url('https://fonts.googleapis.com/css2?family=Prompt:ital@1&display=swap');
+
+.grid-wrapper{
+    display: grid;
+    grid-template-columns: 6fr 4fr;
+    grid-template-rows: auto auto 1fr auto;
+}
+.navbar{
+    grid-column: 1/3;
+    grid-row: 1/2;
+}
+.cart-count{
+    grid-column: 1/3;
+    grid-row: 2/3;
+    padding: 1rem;
+    text-align: center;
+}
+.main{
+    grid-column: 1/2;
+    grid-row: 3/4;
+    margin-left: auto;
+    height: 80vh;
+    margin-right: auto;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+}
+.aside{
+    grid-column: 2/3;
+    grid-row: 3/4;
+    max-width: 20rem;
+    margin-right: auto;
+    margin-left: 1rem;
+    height: fit-content;
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+    padding: 1rem;
+}
+.border-bottom{
+    border-bottom: 1px solid var(--color-light-gray);
+}
+.border-top{
+    border-top:  1px solid var(--color-light-gray);
+}
+.btn-order{
+    width: 100%;
+    margin: 0rem;
+    border-radius: 0rem;
+}
+.text-space-bw{
+    display: flex;
+    justify-content: space-between;
+}
+footer{
+    grid-column: 1/4;
+    grid-row: 5/6;
```

Comment:
May I now why you used grid-row here. I have verified it doesn't making any impact in your footer. Once check and remove if not required.

Reply:
By using both `grid-row` and `grid-column` I have tried being explicit. Will not that help in keeping things clear?

Reply:
Ok that's fine